syntax/arrai.wbnf

```diff
@@ -80,7 +81,7 @@ SEQ_COMMENT -> "," C*;
   | C* odelim="[" C* array=(%!sparse_sequence(top)?) C* cdelim="]" C*
   | C* odelim="<<" C* bytes=(item=(STR|NUM|CHAR|IDENT|"("top")"):SEQ_COMMENT,?) C* cdelim=">>" C*
   | C* odelim="(" tuple=(pairs=(extra | ((rec="rec"? name| name?) ":" v=top)):SEQ_COMMENT,?) cdelim=")" C*
-  | C* odelim="(" identpattern=IDENT cdelim=")" C*
+  | C* odelim="(" identpattern=IDENT_NODOT cdelim=")" C*
```

Comment:
This won't fix `(x)`, which should also be treated as an expression, not an `IdentExpr`. I suspect this line needs to disappear entirely.

Reply:
But I got an example in let_expr_test.go:
```
AssertCodeErrors(t, `let a = 5; let (a) = 1 + 2; a`, "")
```
It means `let a = 5; let (a) = 1 + 2; a` will raise following errors:
```
$ arrai e 'let a = 5; let (a) = 1 + 2; a'
INFO[0000] a is redefined differently 5 vs 3

.:1:12:
let a = 5; let (a) = 1 + 2; a 
```
in this case, `(a)` is parsed to an `IdentExpr` and it is expected behavior.

Reply:
The following code should work:
```arrai
let a = 5; (a)
```